tests/test_sign_edge.py

```diff
@@ -0,0 +1,43 @@
+from faker import Faker
+from hathorlib.scripts import P2PKH
+from hathorlib.utils import get_address_from_public_key_hash, get_hash160
+
+from app_client.transaction import ChangeInfo, TxInput, TxOutput
+from utils import fake_path, fake_tx
+
+fake = Faker()
+
+
+def test_last_chunk_only_change(cmd, public_key_bytes):
+    """
+    This test creates a transaction that will be sent on 2 chunks.
+    The last chunk will have 1 partial and 2 complete outputs, all 3 are change
+    The change list is [0, 2, 3, 4] because we need the last output of the
+    first chunk (output index 1) to be confirmed manually, this would trigger
+    the auto confirmation of the next chunk.
+    This is described on issue #68
+    https://github.com/HathorNetwork/hathor-ledger-app/issues/68
```

Comment:
This is the explanation of how to trigger the issue being solved here.
This test was created on dev so it always failed and then moved to this branch so I can check that it always works.
In doing so I also found the edge case that using `check_output_index_state` fixes.

The values, tx_ids and scripts are randomly generated with Faker but the serialized value will have the same length, so the test will be consistent on this case.